src/app/containers/PageContainer/components/DefaultHeaderComponent/DefaultHeaderComponent.tsx

```diff
@@ -56,6 +57,18 @@ export const DefaultHeaderComponent: React.FC = () => {
     </button>
   );
 
+  const perpetualPage = {
```

Comment:
This doesn't work correctly now, it adds an additional separator with nothing in it on mainnet. You need to filter `pages` (see the comment there).

<img width="940" alt="image" src="https://user-images.githubusercontent.com/14878713/166204434-1e3a1cdd-0c11-4982-b6fb-a63dd63f33d8.png">